tests/test_many_pings.py

```diff
@@ -0,0 +1,63 @@
+from python_on_whales import docker
+from utils import set_netem, Cluster, Client, Server
+import time
+import pytest
+import logging
+
+LOGGER = logging.getLogger(__name__)
+
+
+def linreg(X, Y):
```

Comment:
Maybe you could replace it with
https://numpy-ml.readthedocs.io/en/latest/numpy_ml.linear_models.lm.html#linearregression
https://towardsdatascience.com/linear-regression-in-6-lines-of-python-5e1d0cd05b8d
?